### PR TITLE
Added a "getting started" section to site

### PR DESCRIPTION
**Pull request by VickySteeves**
Refers to issue #103 

Added a section to get users started with some more explicit instructions/definitions/etc. for the proof of concept phase of the project with gifs! Hope this helps!

### CODE REVIEW

**Comment by blahah** [comment]
thanks for this @VickySteeves! Looking at the gifs I think there are a couple of bugs in the version you're running:

1. There is an extra title bar saying 'index.html' - the buttons and the sciencefair logo should be in the same bar. This hasn't been reported before.
2. Some enriched terms in the barplot has XML tags in them. This should be fixed in the latest release `v1.0.5` but you'll need to delete `~/.sciencefair`.

What OS are you running?

File: index.html [hunk 1]
```diff
@@ -123,50 +126,76 @@ <h1 class="w-100 dark-gray f2 tc">
...
+    <img src="assets/discovery.gif" class="w-90-ns">
+  </div>
+  <div class="dtc-ns tc pv4">
+    <h2 class="light-gray">2. Exploring the Lit</h2>
+    <p class="f3 lh-copy ph5-ns light-gray center">ScienceFair has an in-app reader, putting key aspects of articles such as references and figures into the sidebar for convenience.</p>
+    <img src="assets/reading.gif" class="w-90-ns">
+  </div>
+  <div class="dtc-ns tc pv4">
+    <h2 class="light-gray">3. Tagging</h2>
+    <p class="f3 lh-copy ph5-ns light-gray center">ScienceFair gives users the freedom to bulk edit metadata, as well as tag materials with their own keywords for quicker, easier access.</p>
```
Comment (blahah) [inline index.html]: We don't actually provide metadata editing - but we do visualise it. Maybe replace 'edit' with 'visualise'?

Comment (VickySteeves) [inline index.html]: ok, visualize is do-able!

**Comment by VickySteeves** [comment]
I'm running Ubuntu 16.04. The index.html is always here when in the sciencefair app, in the title bar area:

screenshot from 2017-06-30 11-45-04

I'll upgrade and get back to you!

**Comment by blahah** [comment]
Btw I think we should still merge these gifs, but I would also like to address the bugs in your installation and later we can generate gifs that don't show the bugs :)

**Comment by blahah** [comment]
sorry @VickySteeves, it took me a while to realise this was a PR against the `gh-pages` branch. That branch is auto-generated by the code in [`/site`](https://github.com/codeforscience/sciencefair/tree/master/site) - so I am going to migrate these commits to a new branch and merge that (that way I hope you'll still be credited in the commit history)

**Comment by blahah** [comment]
@VickySteeves just checking in to say I haven't forgotten this, just need to get my head around the merge! :)

**Comment by blahah** [comment]
(also we have been racing on bugs - once those are done I can give this my full attention)

**Pull request closed without merging**